src/anupSJun20/PrimeNumber.java

```diff
@@ -0,0 +1,53 @@
+package anupSJun20;
+/*Program 1: Create a class to satisfy below requirements. 
+                  a) Given number is prime or not. 
+                  b) Print total count of prime numbers between 2->100.
+                  c) Find sum of all prime numbers between 2->100
+                  d) Find avg of prime numbers between 2->100*/
+public class PrimeNumber {
+	static int findPrimeNumber(int num) {
+		for(int index = 2;index<=Math.sqrt(num);index++) {
+			if(num%index == 0) {
+				return 0;
+			}
+		}
+		return num;
+	}
+	static int totalCountOfPrimeNumbers(int startNum, int endNum) {
+		int count = 0;
+		for(int index = startNum;index<=endNum;index++) {
+			if (findPrimeNumber(index) != 0)
+				count++;
+		}
+		return count;
+	}
+	
+	static int sumOfPrimeNumbers(int startNum, int endNum) {
+		int sum = 0;
+		for(int index = startNum;index<=endNum;index++) {
+			if (findPrimeNumber(index) != 0)
+				sum = sum + findPrimeNumber(index);
+		}
+		return sum;
+	}
+	static void findaverageofPrimeNumbers(int startNum, int endNum) {
```

Comment:
No need to write extra methods, Just create two instance variable sum and count and perform that operation in isPrime method and use those variable in method like:
void findPrimeNumberInGivenRange(int startNumber, int endNumber) {
for(int index=startNumber; index<=endNumber; index++) {
  findPrimeNumber(index);
}
System.out.println("Total count is: "+count);
System.out.println("Sum of all Prime number is: "+sum);
System.out.println("Average of all prime number is: "+sum/count)

Also you can avoid return statement if you want. Set logic as you want.